src/System.Drawing.Common/src/System/Drawing/Graphics.Windows.cs

```diff
@@ -860,29 +860,23 @@ public void DrawBezier(Pen pen, Point pt1, Point pt2, Point pt3, Point pt4)
         /// <summary>
         /// Draws a series of cubic Bezier curves from an array of points.
         /// </summary>
-        public void DrawBeziers(Pen pen, Point[] points)
+        public unsafe void DrawBeziers(Pen pen, Point[] points)
         {
             if (pen == null)
-            {
                 throw new ArgumentNullException(nameof(pen));
-            }
 
             if (points == null)
-            {
                 throw new ArgumentNullException(nameof(points));
-            }
 
-            IntPtr buf = SafeNativeMethods.Gdip.ConvertPointToMemory(points);
-            try
+            fixed (Point* p = points)
             {
-                int status = SafeNativeMethods.Gdip.GdipDrawBeziersI(new HandleRef(this, NativeGraphics), new HandleRef(pen, pen.NativePen),
-                                                      new HandleRef(this, buf), points.Length);
+                int status = SafeNativeMethods.Gdip.GdipDrawBeziersI(
+                    new HandleRef(this, NativeGraphics),
+                    new HandleRef(pen, pen.NativePen),
+                    new HandleRef(this, new IntPtr(p)), points.Length);
```

Comment:
Do you plan to also address use of HandleRef?  That's an old pattern prior to SafeHandles.

Reply:
> Do you plan to also address use of HandleRef? 

I'm cognizant about it. I'm not sure that I'll deal with it myself, but I'll at least open a follow-up issue.

Reply:
I'll at least clean out the HandleRef around the pinned arrays in the initial work.